### PR TITLE
Add api schema validation for oas

### PR DESCRIPTION
**Pull request by LG-LopezBC**
Before submitting a pull request, please read
https://github.com/bancolombia/scaffold-clean-architecture/wiki/Contributing

## Description
Nowadays most APIs are documented with Open API Specification (OAS) either using the 2.x.x version or 3.x.x version of this specification.

When is builded the API acceptance test, is not possible to do schema validation to the API based on its OAS.

Describe the solution you'd like
Add to the API acceptance test the implementation of a third party library that supports the schema validation based on its OAS.

## Category
- [x] Feature
- [ ] Fix

## Checklist
- [x] The pull request is complete according to the [guide of contributing](https://github.com/bancolombia/scaffold-clean-architecture/wiki/Contributing)
- [x] Automated tests are written
- [x] The documentation is up-to-date
- [x] The pull request has a descriptive title that describes what has changed, and provides enough context for the changelog
- [ ] If the pull request has changed structural files, you have implemented an UpgradeAction according to the [guide](https://github.com/bancolombia/scaffold-clean-architecture/wiki/Contributing#upgradeaction)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -426,11 +426,11 @@ The **`generatePipeline | gpl`** task will generate CI pipeline inside the folde
...
  ## Generate Acceptance Test
  
-The **`generateAcceptanceTest | gat`** task will generate subproject by [karate framework](https://github.com/intuit/karate)  inside the folder "./deployment/",this task does not have  required parameters.
+The **`generateAcceptanceTest | gat`** task will generate subproject by [karate framework](https://github.com/intuit/karate)  inside the folder "./deployment/", this task has one required parameter `name`.
    
 ```shell
-      gradle generateAcceptanceTest
-      gradle gat 
+      gradle generateAcceptanceTest --name=[acceptanceTestProjectName]
+      gradle gat --name=[acceptanceTestProjectName]
```
Comment (santitigaga) [inline README.md:435]: why required? couldn't you have a default value and make it an optional parameter?

File: src/main/java/co/com/bancolombia/task/GenerateAcceptanceTestTask.java [hunk 2]
```diff
@@ -1,18 +1,28 @@
...
 import org.gradle.api.tasks.TaskAction;
+import org.gradle.api.tasks.options.Option;
 
 public class GenerateAcceptanceTestTask extends CleanArchitectureDefaultTask {
+  private String name;
+
+  @Option(option = "name", description = "Set driven adapter name")
+  public void setName(String name) {
+    this.name = name;
+  }
```
Comment (santitigaga) [inline src/main/java/co/com/bancolombia/task/GenerateAcceptanceTestTask.java:17]: it would be more usable if this parameter were optional.

**Review by santitigaga — CHANGES_REQUESTED** [review]

File: src/main/java/co/com/bancolombia/task/GenerateAcceptanceTestTask.java (hunk 2 above)
Comment (LG-LopezBC) [reply src/main/java/co/com/bancolombia/task/GenerateAcceptanceTestTask.java:17]: Ok, following your recommendation, now the parameter name in gat task is optional

File: README.md (hunk 1 above)
Comment (LG-LopezBC) [reply README.md:435]: Sure, I set as a default name for the acceptance test project the value acceptanceTest, now the name parameter is optional

**Review by santitigaga — APPROVED** [review]

**Comment by santitigaga** [comment]
Great Job @LG-LopezBC

**Pull request merged**